RichEditorView/Assets/editor/style.css

```diff
@@ -47,6 +53,9 @@ body, h1, p {
 
 #editor {
     min-height: 100%;
+    height: 100%;
+    overflow: auto;
+    display: block;
 	font-family: 'HelveticaNeue';
```

Comment:
it looks like it wasn't your fault, but could you get this indention to be the same?